src/components/Tooltip/Tooltip.style.ts

```diff
@@ -0,0 +1,63 @@
+import { rem } from 'polished';
+import { Theme } from '../../theme';
+import { css, SerializedStyles } from '@emotion/core';
+import { pickTextColorFromSwatches } from '../../theme/palette';
+import { TooltipSize } from './Tooltip';
+
+export const tooltipStyle = ({ size }: { size: TooltipSize }) => (
+  theme: Theme
+): SerializedStyles => {
+  const color = theme.overrides.tooltip.background.color;
+  const shade = theme.overrides.tooltip.background.shade;
+  const backgroundColor = theme.utils.getColor(color, shade);
+
+  const defineFontSizeBasedOnTooltipSize = (size: TooltipSize) => {
+    if (size === 'large') {
+      return theme.typography.fontSizes['16'];
+    } else if (size === 'small') {
+      return theme.typography.fontSizes['12'];
+    }
+
+    return theme.typography.fontSizes['14'];
+  };
+
+  return css`
```

Comment:
you can avoid all the `!important` by adding a new parent to the tooltip. Example:
jsx
```
   <div css={tooltipStyle({ size, placement })}>
        <ReactTooltip id={tooltipID} place={placement} effect="solid" className="tooltip">
          {content}
        </ReactTooltip>
      </div>
```

styles
```
   > div {
      color: ${pickTextColorFromSwatches(color, shade)};
      background-color: ${backgroundColor};
      max-width: ${rem(256)};
      padding: ${theme.spacing.sm};
      font-size: ${defineFontSizeBasedOnTooltipSize(size)};
      line-height: 110%;
      border-radius: ${theme.spacing.sm};
      text-align: justify;
    }

    .show {
      opacity: 1 !important;
    }
    ....
```

Reply:
You can also replace  `> div` with `> ${tooltipId}` to make it more secure for future changes 😄

Reply:
i though of this but is it really !important ? (see what i did there?)

Reply:
I think it's always !important to avoid `!important;`